src/main.cc

```diff
@@ -434,10 +433,6 @@ load_cfg(void)
 		}
 	}
 
-	/*
-	 * pid file check must happen before logger init in order for the
-	 * error message to show in stderr
-	 */
```

Comment:
It's important not to print anything to the other instance's log in this case so this comment was here for a reason. Please don't change this behavior.